charts/minikube-values.yaml

```diff
@@ -15,23 +15,29 @@ ingress:
 global:
   gitlab:
     clientSecret: e3ecd740-0892-4610-ad69-c7f4886f4533
-    postgresPassword: 3b0aa4bae4fbbe167143f583e35eda18
```

Comment:
It would be much nicer to have all the secrets in this file auto-created. Having all those random strings in here has been bothering me anyway and leaves the impression of Renku being a pain to deploy.